### PR TITLE
telldus-mqtt: add new package in utils

### PR DESCRIPTION
**Pull request by PeterFromSweden**
Extends functionality of exisiting telldus-core package

Maintainer: @PeterFromSweden
Compile tested: bcm2708/RPiB/openwrt23.05.0, mvebu/cortexa9/openwrt23.05.0
Run tested: bcm2708/RPiB/openwrt23.05.0, mvebu/cortexa9/openwrt23.05.0

Description:
Extends functionality of exisiting telldus-core package

### CODE REVIEW

File: utils/telldus-mqtt/Makefile [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=telldus-mqtt
+PKG_VERSION:=0.1
+PKG_RELEASE:=4
+
+# Development hint, use command line
+# make SOURCE_DIR=/home/ubuntu/telldus-mqtt package/telldus-mqtt/compile
+
+ifeq ($(SOURCE_DIR),)
```
Comment (BKPepe) [inline utils/telldus-mqtt/Makefile:15]: ?

File: utils/telldus-mqtt/Makefile [hunk 2]
```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (C) Peter Liedholm
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=telldus-mqtt
+PKG_VERSION:=0.1
+PKG_RELEASE:=4
```
Comment (BKPepe) [inline utils/telldus-mqtt/Makefile:10]: Always starts with 1

File: utils/telldus-mqtt/Makefile [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+ifneq ($(SOURCE_DIR),)
+# Only used for SOURCE_DIR
+# Package preparation instructions; create the build directory and copy the source code. 
+# The last command is necessary to ensure our preparation instructions remain compatible with the patching system.
+define Build/Prepare
+		mkdir -p $(PKG_BUILD_DIR)
+		cp -r $(SOURCE_DIR)/* $(PKG_BUILD_DIR)
+		$(Build/Patch)
+endef
+endif
```
Comment (BKPepe) [inline utils/telldus-mqtt/Makefile:49]: Why? Do we really need this? Isnt it default?

File: utils/telldus-mqtt/Makefile [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+define Package/telldus-mqtt/install
+	$(INSTALL_DIR)  $(1)/usr/bin
+	$(INSTALL_BIN)  $(PKG_INSTALL_DIR)/usr/bin/telldus-mqtt $(1)/usr/bin
+	$(INSTALL_DIR)  $(1)/etc/telldus-mqtt/
+	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/usr/etc/telldus-mqtt/telldus-mqtt.json $(1)/etc/telldus-mqtt
```
Comment (BKPepe) [inline utils/telldus-mqtt/Makefile:55]: Shouldnt it be marked as configuration file?

File: utils/telldus-mqtt/Makefile [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/usr/etc/telldus-mqtt/telldus-mqtt-homeassistant.json $(1)/etc/telldus-mqtt
```
Comment (BKPepe) [inline utils/telldus-mqtt/Makefile:56]: Home Assistant? Is it really useful somehow?

File: utils/telldus-mqtt/files/telldus-mqtt [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+start_service() {
+	procd_open_instance
+	procd_set_param command $PROG
+	procd_append_param command --nodaemon # foreground required by procd
+	procd_set_param file $CONFFILE
+	procd_set_param respawn  # respawn the service if it exits
+	procd_set_param stdout 1 # forward stdout of the command to logd
+	procd_set_param stderr 1 # same for stderr
+	procd_close_instance
+}
```
Comment (BKPepe) [inline utils/telldus-mqtt/files/telldus-mqtt:18]: New line here.

**Review by BKPepe — CHANGES_REQUESTED** [review]

File: utils/telldus-mqtt/Makefile (hunk 5 above)
Comment (PeterFromSweden) [reply utils/telldus-mqtt/Makefile:56]: Yes, but this only relates to Home Assitant MQTT topic syntax for autodetection. It is not required for MQTT in general.

File: utils/telldus-mqtt/Makefile (hunk 1 above)
Comment (PeterFromSweden) [reply utils/telldus-mqtt/Makefile:15]: Development stuff. Deleted.

File: utils/telldus-mqtt/Makefile (hunk 2 above)
Comment (PeterFromSweden) [reply utils/telldus-mqtt/Makefile:10]: Sure. Fixed.

File: utils/telldus-mqtt/Makefile (hunk 3 above)
Comment (PeterFromSweden) [reply utils/telldus-mqtt/Makefile:49]: Development stuff. Deleted.

File: utils/telldus-mqtt/files/telldus-mqtt (hunk 6 above)
Comment (PeterFromSweden) [reply utils/telldus-mqtt/files/telldus-mqtt:18]: end of line added at EOF.

**Comment by PeterFromSweden** [comment]
@BKPepe all feedback incorporated.

**Pull request merged**